packages/story-editor/src/elements/video/output.js

```diff
@@ -54,19 +54,15 @@ function VideoOutput({ element, box }) {
 
   return (
     <MediaOutput element={element} box={box} data-leaf-element="true">
-      <amp-video {...videoProps} id={`el-${element.id}-media`}>
+      <amp-video
+        {...videoProps}
+        id={`el-${element.id}-media`}
+        // Actual <amp-story-captions> output happens in OutputPage.
+        captions-id={
+          tracks?.length > 0 ? `el-${element.id}-captions` : undefined
+        }
+      >
```

Comment:
Should we feature flag this? I'm not sure what the test coverage or stability of amp-story-captions is since it's so new.

Reply:
Yeah not sure about that one, I hesitated a bit. Upstream test coverage is [arguably not perfect](https://github.com/ampproject/amphtml/pull/35682), but it seems pretty stable to me so far and no bugs have been reported yet.

A flag might be useful if we need continuous iteration on the styling.